apps/frontend/src/pages/profile/index.tsx

```diff
@@ -135,6 +136,9 @@ const ProfilePage: React.FC = () => {
                     <span className="text-[14px]">Add Chip</span>
                   </AppButton>
                 </div>
+                <div className="w-fit">
```

Comment:
Can this be near the bottom (near dark theme and logout) instead?

Reply:
Fixed